.github/workflows/My-CI-Pipeline.yml

```
@@ -0,0 +1,17 @@
+name: Continuous Integration
+on:
+  pull_request:
+    types: [opened, reopened]
+  push: [push]
+  paths: 
+    - 'README.md'
```

Note that it looks like these didn't quite work, the following run shows an error:
https://github.com/Dani2525/Devops-Course-Starter/actions/runs/2536794569

I think you're probably looking for:
```suggestion
  pull_request:
    types: [opened, reopened]
    paths-ignore: 
      - 'README.md'

  push:
    paths-ignore: 
      - 'README.md'
```